scripts/bash/s3a_wrapper.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+# Copyright 2018 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# We have 2 states.
+#   Write Standardisation
+#   Write Conformance
+# The script will have to figure out which is being run and if it is being written to S3 or HDFS.
+#   In case of standardisation parsing the java properties supplied to figure out the standardisation path.
+#   In case of conformance calling Menas for info over REST API using supplied metadata
+# Then cleaning up the paths to conform to the path schema in query provided.
+
+# TODO - Auto load jceks files
```

Comment:
When this is planned to be done? Exist Issue already?

Reply:
Now. 😄